features/support/persistent_browser.rb

```diff
@@ -30,31 +30,25 @@ def self.is_remote?(target)
     def self.watir_browser(target)
       if is_remote?(target)
         Watir::Browser.new(:remote,
-                           :http_client => client,
                            :url => url,
                            :desired_capabilities => desired_capabilities)
       else
         browser = target.gsub('local_', '').to_sym
-        Watir::Browser.new browser, :http_client => client
+        Watir::Browser.new browser
       end
     end
 
     def self.selenium_browser(target)
       if is_remote?(target)
         Selenium::WebDriver.for(:remote,
-                                :http_client => client,
                                 :url => url,
                                 :desired_capabilities => desired_capabilities)
       else
         browser = target.gsub('local_', '').to_sym
-        Selenium::WebDriver.for browser, :http_client => client
+        Selenium::WebDriver.for browser
       end
     end
 
-    def self.client
-      Selenium::WebDriver::Remote::Http::Persistent.new
```

Comment:
This client blows up when the browser is instantiated. I'm not sure exactly why but it seems like the `start` method has changed.
https://travis-ci.org/cheezy/page-object/jobs/569885122#L645

It looks like this class is marked private and shouldn't be created directly:
https://seleniumhq.github.io/selenium/docs/api/rb/Selenium/WebDriver/Remote/Http/Persistent.html

Reply:
Persistent client is no longer necessary. Current Selenium default client provides keep-alive functionality. But... adopting this code should require minimum Selenium to be bumped to 3.142.3

Reply:
done